smdebug/tensorflow/collection.py

```diff
@@ -108,7 +110,10 @@ def get_export_names_of_tensors(self):
         return self.tensor_names
 
     def get_tensor(self, name):
-        return self._tensors[name]
+        if name in self._tensors:
+            return self._tensors[name]
+        else:
+            return None
```

Comment:
Could you check if returning None causes issues wherever this function is used? I don't think None is expected. Is it actually possible? Did you see it happening?